erpnext/maintenance/doctype/maintenance_schedule_detail/maintenance_schedule_detail.json

```diff
@@ -57,30 +33,40 @@
    "report_hide": 1
   },
   {
-   "fieldname": "sales_person",
+   "fieldname": "project_template",
    "fieldtype": "Link",
    "in_list_view": 1,
-   "label": "Sales Person",
-   "oldfieldname": "incharge_name",
-   "oldfieldtype": "Link",
-   "options": "Sales Person",
+   "label": "Project Template",
+   "options": "Project Template",
    "reqd": 1
```

Comment:
How about we make project_template non mandatory for manual entries ... also use the project_template_query in client script